rwt_app_chooser/README.md

```diff
@@ -0,0 +1,111 @@
+rwt_app_chooser
+===============
+
+App chooser client for web browsing device
+
+### Usage
+
+1. (On Robot) Launch app manager
+
+    ```bash
+    roscore &
+    rosparam set robot/type pr2     # set robot type as rosparam
+    rosparam set robot/type pr1012  # set robot name as rosparam
+    roslaunch rwt_app_chooser app_manager.launch
+    ```
+    
+    You can use custom installed app list by:
+    
+    ```bash
+    roslaunch rwt_app_chooser app_manager.launch app_list:=/path/to/applist_dir
+    ```
+
+2. (On Server) Launch rwt_app_chooser server
+
+    ```bash
+    roslaunch rwt_app_chooser rwt_app_chooser.launch
+    ```
+    
+    This can be run on the robot
+    You can now access to `http://<server ip>:8080/`
```

Comment:
@furushchev 8080 -> 8000 ?

Reply:
or change https://github.com/tork-a/visualization_rwt/pull/67/files#diff-02f89977d9249098d63ca5fedd1fd199R3

Reply:
@k-okada Ok, it's also a typo... I'll update soon...